docs/manual/docs/about/release-notes.md

```diff
@@ -23,7 +23,7 @@
 ### New commands
 
 **SharePoint Online:**
-
+- [spo folder move](../cmd/spo/folder/folder-move.md) - moves a folder to another location [#672](https://github.com/pnp/office365-cli/issues/424)
```

Comment:
In the future, there is no need to update the release notes, because you can't know upfront when the PR will be processed and in which release it will be included. I appreciate the help though 👍